sssd_test_framework/roles/samba.py

```diff
@@ -1061,6 +1097,97 @@ def policy(self, logon_rights: dict[str, list[SambaObject]], cfg: dict[str, Any]
         return self
 
 
+class SambaPasswordPolicy(GenericPasswordPolicy):
+    """
+    Password policy management.
+    """
+
+    def __init__(self, role: Samba):
+        """
+        :param role: Samba host object.
+        :type role: SambaHost
+        """
+        super().__init__(role)
+
+    def complexity(self, enable: bool) -> SambaPasswordPolicy:
+        """
+        Enable or disable password complexity.
+
+        :param enable: Enable or disable password complexity.
+        :type enable: bool
+        :return: SambaPasswordPolicy object.
+        :rtype: SambaPasswordPolicy
+        """
+        complexity: str = "on" if enable else "off"
+
+        args: CLIBuilderArgs = {
+            "complexity": (self.cli.option.VALUE, complexity),
+        }
+
+        self.host.conn.run(self.cli.command("samba-tool domain passwordsettings set", args))
+
+        return self
+
+    def lockout(self, duration: int, attempts: int) -> SambaPasswordPolicy:
+        """
+        Set lockout duration and login attempts.
+
+        :param duration: Duration of lockout in seconds, converted to minutes.
+        :type duration: int
+        :param attempts: Number of login attempts.
+        :type attempts: int
+        :return: SambaPasswordPolicy object.
+        :rtype: SambaPasswordPolicy
+        """
+        minutes = divmod(duration, 60)[0]
+
+        args: CLIBuilderArgs = {
+            "account-lockout-duration": (self.cli.option.VALUE, str(minutes)),
+            "account-lockout-threshold": (self.cli.option.VALUE, str(attempts)),
+        }
+        self.host.conn.run(self.cli.command("samba-tool domain passwordsettings set", args))
+
+        return self
+
+    def age(self, minimum: int, maximum: int) -> SambaPasswordPolicy:
+        """
+        Set maximum and minimum password age.
+
+        :param minimum: Minimum password age in seconds, converted to days.
```

Comment:
Are converted to days? The function round these values down to hours.

Reply:
Initially I didn't know which one was right, whether the comment or the code. With your explanation, now I know the comment is right. So, my point is that code converts the seconds to hours (3600 seconds), not days (86400 seconds).